traveler_app/src/main/java/com/guestlogix/traveler/settings/EditEndpointPreference.java

```diff
@@ -0,0 +1,28 @@
+package com.guestlogix.traveler.settings;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.webkit.URLUtil;
+
+import androidx.preference.EditTextPreference;
+import androidx.preference.Preference;
+
+public class EditEndpointPreference extends EditTextPreference {
```

Comment:
whats the purpose of this class?

Reply:
ok I get it now...make naming a bit more generic.

`EditUrlPreference` is more generic...not talking about endpoints.

This class has value in ALL projects

Reply:
done